utils/users.js

```diff
@@ -81,9 +81,19 @@ const getParticipantUserIds = async (participantArray) => {
   }
 };
 
+/**
+ * Returns a random object from the array of colors to user
+ * @param array {array} : array containing objects
+ * @returns random Index number : index between the range 0 to array.length
+ */
+const getRandomIndex = (array = []) => {
+  return Math.floor(Math.random() * (array.length - 0) + 0);
+};
+
```

Comment:
Can we put this function on a more generic file? Such as `utils/helper.js`

Reply:
should i make a `migrations.js` file?

Reply:
getRandomIndex is a generic function that can be used anywhere. I don't think migrations.js would be suitable here.

Reply:
got it, made the changes